charts/mercure/values.yaml

```diff
@@ -0,0 +1,129 @@
+# Default values for mercure.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- enable the debug mode
+debug: false
+# -- enable the development mode, including the debug UI and the demo
+dev: false
+# -- the URL representation of the transport to use
+transportUrl: bolt:///data/mercure.db
+# -- extra Mercure directives to include in the Caddyfile
+extraDirectives: ""
+
+# -- the JWT key to use for publishers, a random key will be generated if empty
+publisherJwtKey: ""
+# -- the JWT algorithm to use for publishers
+publisherJwtAlg: HS256
+
+# -- the JWT key to use for subscribers, a random key will be generated if empty
+subscriberJwtKey: ""
+# -- JWT algorithm to use for subscribers
+subscriberJwtAlg: HS256
+
+# -- license key for [the High Availability version](https://mercure.rocks/docs/hub/cluster) (not necessary is you use the FOSS version):
+license: ""
+
+# -- the number of replicas to create, this value must be 1 unless you are using [the High Availability version](https://mercure.rocks/docs/hub/cluster)
+replicaCount: 1
+
+image:
+  repository: dunglas/mercure
+  pullPolicy: IfNotPresent
+  # -- overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # -- specifies whether a service account should be created
+  create: true
+  # -- annotations to add to the service account
+  annotations: {}
+  # -- the name of the service account to use.
+  # -- If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: mercure-example.local
+      paths:
+      - path: /
+        backend:
+          serviceName: mercure-example.local
+          servicePort: 80
+  tls: []
+  #  - secretName: mercure-example-tls
+  #    hosts:
+  #      - mercure-example.local
```

Comment:
This needs to be updated as well.

Reply:
Good catch, thanks! I also shamelessly copied your docs for the values that weren't documented.

Reply:
No problem! 😄